CMakeLists.txt

```diff
@@ -24,61 +27,20 @@ endif ()
 
 find_package(MPI REQUIRED)
 
-# Add include paths
-if (FIBER_ENABLE_HEFFTE)
-add_definitions(-DFIBER_ENABLE_HEFFTE)
-include_directories($ENV{HOME}/heffte/build/include)
-link_directories($ENV{HOME}/heffte/build)
-endif()
-
-if (FIBER_ENABLE_FFTMPI)
-add_definitions(-DFIBER_ENABLE_FFTMPI)
```

Comment:
`add_definitions()` adds a compile-time flag. Now the flag is missing. Is this intended?

Reply:
According to the following documentation, the -D... flags are automatically added to COMPILE_DEFINITIONS.  I have verified that this is the case (using heffte). https://cmake.org/cmake/help/latest/command/add_definitions.html

Reply:
Why you make this comment so many times? ;-)